pkgs/by-name/fr/frankenphp/package.nix

```diff
@@ -0,0 +1,94 @@
+{ lib
+, stdenv
+, buildGoModule
+, fetchFromGitHub
+, php
+, pcre2
+, libargon2
+, libxml2
+, systemd
+, testers
+, frankenphp
+, runCommand
+, writeText
+
+, phpOverrideArgs ? {}
+}:
+
+let
+  phpEmbedWithZts = php.override {
+    embedSupport = true;
+    ztsSupport = true;
+  } // (lib.filterAttrs (n: v: !(builtins.elem n [ "embedSupport" "ztsSupport" ])) phpOverrideArgs);
```

Comment:
I don't think/know if this line is really needed as `embedSupport` and `ztsSupport` will always be enabled, and `phpOverrideArgs` can be replaced by the user as he would override himself the `php` provided package.

Reply:
Yes, I would remove it after giving a second thought.